debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (25.3~3g0cb64272-0ubuntu1) UNRELEASED; urgency=medium
+
+  * d/p/0001-Revert-fix-support-bond-names-in-network_data.patch
```

Comment:
I like the new naming convention, ordering the prefix with a number makes it easier for us to avoid having to pop open debian/patches/series to check on the order of patches being applied.